### PR TITLE
Meta analyses

### PR DESCRIPTION
**Pull request by alibow**
For epi group work :)

### CODE REVIEW

File: docs/source/model_design/general_research.rst [hunk 1]
```diff
@@ -831,11 +831,93 @@ Ecologic Studies
...
+~~~~~~~~~~~~~~~~~~
+
+Meta-analyses can be performed for dichotomous and continuous outcomes. For dichotomous outcomes, the measure of effect is typically risk ratios or odds ratios; for continous outcomes the measure of effect is typically standardized mean differences.
+
+Fixed Versus Random Effects Assumptions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two main hypotheses regarding meta-analysis, referred to as fixed effects and random effects assumptions, discussed below.
+
+In **fixed effects** analysis, it is assumed that the *true* effect size is the same in all studies, although they may vary slightly due to random sampling error. Therefore, a summary meta-analysis effect size in fixed effects analysis will be an *estimate of the* **one common** *effect size.*
```
Comment (aflaxman) [inline docs/source/model_design/general_research.rst:846]: plural "fixed effects" or singular "fixed effect"?

File: docs/source/model_design/general_research.rst [hunk 2]
```diff
@@ -831,11 +831,93 @@ Ecologic Studies
...
+
+  Fixed effects analysis should be chosen if two conditions are met:
+
+    #. It is believed that all studies included in the meta-analysis are functionally identical.
+
+    #. The goal of the meta-analysis is to compute the common effect size for an identified population and is not intended to be generalized to another population.
+
+  An example of an appropriate application of fixed effects analysis might be a drug trial intended to include 1,000 study subjects, but that must be conducted in sub-trials of around 100 study subjects at a time due to resource constraints. Fixed effects analysis may be used to estimate a summary effect of all of the sub-trials.
+
+In **random effects** analysis, it is assumed that the *true* effect size varies from one study to the next due to population heterogeneity. Therefore, a summary meta-analysis in random effects analysis will be an *estimate of the* **mean** *of these effects.*
```
Comment (aflaxman) [inline docs/source/model_design/general_research.rst:858]: same question about singular vs plural for random effects

File: docs/source/model_design/general_research.rst [hunk 3]
```diff
@@ -831,11 +831,93 @@ Ecologic Studies
 Meta-Analyses and Meta-Regressions
 ++++++++++++++++++++++++++++++++++
 
+Meta-Analyses and Meta-Regressions are two similar study designs that allow researchers to combine and summarize findings from several individual studies that investigated the same association between an exposure and outcome. Studies included in meta-analyses and meta-regressions are typically identified using systematic literature searchers (see `Conducting Systematic Literature Searches/Reviews`_) and chosen for inclusion based on a set of inclusion and exclusion criteria for that particular meta-analysis or -regression. While each study in a meta-analysis or meta-regression should have the same exposure and outcome of interest, they may differ with respect to study design, study populations, and confounding bias. Briefly, meta-analysis summarizes study findings without formal control for differences between individual studies while meta-regression allows for flexible formal control for these differences. Details are discussed below.
```
Comment (beatrixkh) [inline docs/source/model_design/general_research.rst:834]: I am still not entirely clear on how to do a meta analyses without meta regression (would you take a weighted average? is there a technique for doing this with random effects without meta regression?) but as far as I understand it, meta-analyses are a type of study, and meta regressions are a tool used in meta analyses, (as opposed to them being similar/distinct study designs).

File: docs/source/model_design/general_research.rst [hunk 4]
```diff
@@ -831,11 +831,93 @@ Ecologic Studies
...
+
+Measures of Effect
+~~~~~~~~~~~~~~~~~~
+
+Meta-analyses can be performed for dichotomous and continuous outcomes. For dichotomous outcomes, the measure of effect is typically risk ratios or odds ratios; for continous outcomes the measure of effect is typically standardized mean differences.
```
Comment (beatrixkh) [inline docs/source/model_design/general_research.rst:839]: What about categorical?

**Pull request merged**